docs/reference/mapping/types/text.asciidoc

```diff
@@ -133,8 +133,12 @@ The following parameters are accepted by `text` fields:
 <<mapping-store,`store`>>::
 
     Whether the field value should be stored and retrievable separately from
-    the <<mapping-source-field,`_source`>> field. Accepts `true` or `false`
-    (default).
+    the <<mapping-source-field,`_source`>> field. Accepts `true` or `false` (default).
+    If <<synthetic-source,synthetic _source>> is enabled, then this
```

Comment:
Maybe remove `If <<synthetic-source,synthetic _source>> is enabled` part? The `store` param defaults to `true` for tsdb indices, when there is no suitable multi field of type keyword .

Reply:
Woops, thanks. This is leftover.